sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -270,12 +273,25 @@ trait CheckAnalysis extends PredicateHelper {
 
         operator match {
           case o if o.children.nonEmpty && o.missingInput.nonEmpty =>
+            val resolver = conf.resolver
+            val commonAttrs = o.missingInput.filter(x =>
```

Comment:
Not quite common attrs actually. Maybe `attrsWithSameName`.

Reply:
Indeed!